### ISSUE TITLE
Data is not visible in the gantt chart

### ISSUE THREAD

**Issue by AshishKrGoya1** [cerner/terra-core#3881]
## Description
Data is not showing up in the label truncation. It would be good to see sample data to visualise data in this particular gantt chart

![image](https://github.com/cerner/terra-core/assets/142099654/5cfb552d-1c12-42c1-929b-f562a80b6615)
